test/export/test_passes.py

```diff
@@ -234,6 +234,18 @@ def forward(self, x):
         self.assertTrue(pass_result.modified)
         self.assertEqual(count_call_function(pass_result.graph_module.graph, torch.ops.aten.view.default), 0)
 
+    def test_functionalization_with_view_copy(self) -> None:
+        def foo(x):
+            x.add_(4)
+            y = x.view(x.shape)
+            return x.cos() + y.cos()
+
+        x = torch.zeros(4, 2, 3)
+
+        ep = export(foo, (x,)).transform(ReplaceViewOpsWithViewCopyOpsPass())
+        # After this pass, there shouldn't be any view nodes in the graph
+        self.assertTrue(count_call_function(ep.module.graph, torch.ops.aten.view_copy.default) > 0)
```

Comment:
Can you check that there are zero view nodes instead of checking the existence of view_copy nodes?